libs/providers/flagd/src/lib/service/grpc/service.ts

```diff
@@ -186,7 +209,7 @@ export class GRPCService implements Service {
       this._streamConnectAttempt++;
       setTimeout(() => {
         this._streamConnectBackoff = this._streamConnectBackoff * 2;
-        this.connectStream();
+        this.connectStream(connectCallback, changedCallback, disconnectCallback);
```

Comment:
Should this be awaited?

Reply:
I don't see why, the function just has side effects, and it's set on a timeout.